component/tests/golden/defaults/defaults/defaults/cronjob.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  labels:
+    app.kubernetes.io/component: exoscale-metrics-collector
+    app.kubernetes.io/managed-by: commodore
+    app.kubernetes.io/name: exoscale-metrics-collector
+    app.kubernetes.io/part-of: appuio-cloud-reporting
+  name: defaults
+  namespace: appuio-cloud-reporting
+spec:
+  concurrencyPolicy: Forbid
+  failedJobsHistoryLimit: 5
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - args:
+                - exoscale-metrics-collector
+              command:
+                - sh
+                - -c
+              env:
+                - name: password
+                  valueFrom:
+                    secretKeyRef:
+                      key: password
+                      name: reporting-db
+                - name: username
+                  valueFrom:
+                    secretKeyRef:
+                      key: username
+                      name: reporting-db
+                - name: ACR_DB_URL
+                  value: postgres://$(username):$(password)@reporting-db.appuio-reporting.svc:5432/reporting?sslmode=disable
+                - name: EXOSCALE_API_KEY
+                  valueFrom:
+                    secretKeyRef:
+                      key: api_key
+                      name: cronjob-defaults
+                - name: EXOSCALE_API_SECRET
+                  valueFrom:
+                    secretKeyRef:
+                      key: api_secret
+                      name: cronjob-defaults
+                - name: K8S_SERVER_URL
+                  valueFrom:
+                    secretKeyRef:
+                      key: cluster_server
+                      name: cronjob-defaults
+                - name: K8S_TOKEN
+                  valueFrom:
+                    secretKeyRef:
+                      key: cluster_token
+                      name: cronjob-defaults
```

Comment:
This seems to be wrong? Is this file from an older gen-golden that shouldn't be here anymore?

Reply:
yes I renamed the defaults. I will remove the whole folder.